docs/items/weapon-configs.md

```diff
@@ -368,12 +376,16 @@ What values a Weapon Type can have:
     * Direct Damage Scalar: 2.10
     * Grenade Damage Scalar: 1.00
     * Explosive Damage Scalar: 1.00
+  * VFX: Infected Alpha
```

Comment:
The VIP and Infected VFX aren't traits so lets put these under the scripted behavior